apps/condo/domains/ticket/components/TicketChanges/TicketChange.tsx

```diff
@@ -316,11 +333,88 @@ const useChangedFieldMessagesOf = (ticketChange) => {
             )
         }
     }
+
+    const formatAutoReopenDiffMessage = (field, ticketChange) => {
+        if (field === 'deferredUntil') {
+            const valueDeferredUntilFrom = formatField(field, ticketChange['deferredUntilFrom'], TicketChangeFieldMessageType.From)
+
+            return (
+                <FormattedMessage
+                    id='pages.condo.ticket.TicketChanges.autoReopenTicket.deferredUntil'
+                    values={{
+                        deferredUntil: valueDeferredUntilFrom,
+                    }}
+                />
+            )
+        } else if (field === 'statusDisplayName') {
```

Comment:
Too many else if for one field conditions
You can use object with keys for field value 
Or switch ... case